src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -989,7 +989,24 @@ private void Dispose()
             if (String.IsNullOrWhiteSpace(reason) && !this.EntrypointHit)
             {
                 breakRequest = BreakRequest.None;   // don't let stopping interfere with launch processing
-                this.EntrypointHit = true;
+                
+                // MinGW sends a stopped event on attach. gdb<->gdbserver also sends a stopped event when first attached.
+                // If this is a gdb<->gdbserver connection, ignore this as the entryPoint
+                if (this._launchOptions is LocalLaunchOptions &&
+                    !String.IsNullOrWhiteSpace(((LocalLaunchOptions)this._launchOptions).MIDebuggerServerAddress))
+                {
+                    // If the stopped event occurs on gdbserver, ignore it unless it contains a filename.
+                    TupleValue frame = results.Results.TryFind<TupleValue>("frame");
+                    if (frame.Contains("file"))
```

Comment:
I am a little confused by this part since aren't we going to continue from this event no matter what on line 993?